sdk/core/azure-core/src/main/java/com/azure/core/http/policy/UserAgentPolicy.java

```diff
@@ -21,7 +21,7 @@
 
     // From the design guidelines, the default user agent header format is:
     // azsdk-java-<client_lib>/<sdk_version> <platform_info>
-    private static final String USER_AGENT_FORMAT = DEFAULT_USER_AGENT_HEADER + "-%s/%s %s";
+    private static final String USER_AGENT_FORMAT = DEFAULT_USER_AGENT_HEADER + "-%s/%s (%s)";
```

Comment:
Can you please add a unit test to ensure that the generated string is as expected and future changes don't accidentally break this format?

Reply:
updated